test/specs/utils/error.js

```diff
@@ -66,4 +66,24 @@ describe('Custom Errors', () => {
 			);
 		},
 	);
+	Given(
+		'a function that throws a print error "cannot be displayed"',
+		given.aFunctionThatThrowsAPrintError,
+		() => {
+			When('the print error is thrown', when.thePrintErrorIsThrown, () => {
+				Then(
+					'it should print the error message in red',
+					then.itShouldPrintTheErrorMessageInRed,
+				);
+				Then(
+					'the error should have the name "PrintError"',
+					then.theErrorShouldHaveTheName,
+				);
+				Then(
+					'the error should be an instance of Node’s built-in Error',
+					then.theErrorShouldBeInstanceOfNodesBuiltInError,
```

Comment:
I realise this was here before, but can you update the name of this function to match the descriptions?